app/views/sulten/reservations/admin_new.html.haml

```diff
@@ -14,7 +14,7 @@
     = f.input :reservation_duration, label: t("sulten.reservation.form.duration"), collection: [30, 60, 90, 120, 180], member_label: proc{|d| "#{d} minutter"}, selected: 120
     = f.input :name, label: t("sulten.reservation.name")
     = f.input :reservation_type, label: t("sulten.reservation.reservation_type")
-    = f.input :telephone, label: t("sulten.reservation.telephone")
+    = f.input :telephone, label: t("sulten.reservation.telephone"), required: false, input_html: { disabled: true, style: "opacity: 0.5;" }
```

Comment:
Wouldn't it be better to just delete this line?

Reply:
+1